src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -380,7 +380,8 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
               snapshotService,
               datasetId,
               datasetName,
-              snapshotId));
+              snapshotId),
+          randomBackoffRetry);
```

Comment:
I saw the retry step status in the step itself, what does this do?

Reply:
Stairway requires both in order to work:
- the randomBackoffRetry defines _how_ to perform the retries (backoff time, # of retries)
- the try/catch defines _when_ the retries occur - particularly on which exception we want to retry